aemsites/888de
# Footer bar fixes

Fixed so <a> wraps block
Fixed mobile styles - Iphone SE

## Ticket/Issue number(s)
Fix 184
Fix 162

## Test URLs
- Before: https://main--888de--aemsites.hlx.page/
- After: https://footer-bar-fixes--888de--aemsites.hlx.page/

## Testing instructions
1. Make sure footer bar is clickable and url shown upon hovering
2. Make sure mobiles styles are good - specifically Iphone SE
